docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -594,8 +594,18 @@ If you are using the `quarkus-micrometer` or `quarkus-smallrye-metrics` extensio
 This behavior must first be enabled by setting the `quarkus.mongodb.metrics.enabled` property to `true` in your `application.properties`.
 
 So when you access the `/q/metrics` endpoint of your application you will have information about the connection pool status.
-When using link:microprofile-metrics[SmallRye Metrics], connection pool metrics will be available under the `vendor` scope.
+When using link:smallrye-metrics[SmallRye Metrics], connection pool metrics will be available under the `vendor` scope.
```

Comment:
I just corrected the wrong links.
If micrometer is supported for MongoDB metrics, we should switch the all sentence for micrometer

Reply:
Yes, exactly. Not sure if Mongo has migrated yet.

Reply:
@ebullient I propose to merge this as is, it fixes the bad link.
We can document (you or me) later micrometer instead of smallrye (or let both for the moment, I don't know what is the policy for metrics).

Reply:
Reading the all section again, this is OK, it talks about both metrics impl.
This sentence if for a specificity of the metrics for MP metrics.